tools/rocqwc.mll

```diff
@@ -161,13 +161,28 @@ and proof = parse
   | "Proof" space* '.'
   | "Proof" space+ "with"
   | "Proof" space+ "using"
-           { seen_proof := true; proof lexbuf }
+           { seen_proof := true; started_proof lexbuf }
   | "Proof" space
            { proof_term lexbuf }
   | proof_end
            { seen_proof := true; spec lexbuf }
   | character | _
```

Comment:
```suggestion
  | word | character | _
```

where `word = ['A'-'Z' 'a'-'z' '0'-'9' '_']+`

(I am suggesting to do this instead of `started_proof`.)

Reply:
not quite, because `character` is the regex for character literals.

Reply:
>word = ['A'-'Z' 'a'-'z' '0'-'9' '_']+

shouldn't we use something more unicode-supporting? αProof shouldn't be treated as Proof either.

Reply:
Is αProof something you just came up with or did you see it in the wild? :)

I wouldn't push the extra work on someone, but if anyone wants to do it I would accept it.